modules/kmm-hub-using-the-managedclustermodule.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-hub-using-the-managedclustermodule_{context}"]
+= Using the `ManagedClusterModule` CRD
+
+Use the `ManagedClusterModule` Custom Resource Definition (CRD) to configure the deployment of kernel modules on spoke clusters.
+This CRD is cluster-scoped, wraps a ``Module`` spec and adds the following additional fields:
+
+[source,yaml]
+----
+apiVersion: hub.kmm.sigs.x-k8s.io/v1beta1
+kind: ManagedClusterModule
+metadata:
+  name: <my-mcm>
+  # No namespace, because this resource is cluster-scoped.
+spec:
+  moduleSpec: <1>
+    selector: <2>
+      node-wants-my-mcm: 'true'
+
+  spokeNamespace: <some-namespace> <3>
+
+  selector:
+    wants-my-mcm: 'true'  # Selects ManagedCluster objects
+----
+<1> `moduleSpec`: Contains `moduleLoader` and `devicePlugin` sections, just like in a `Module` resource.
```

Comment:
```suggestion
<1> Contains the `moduleLoader` and `devicePlugin` sections, similar to a `Module` resource.
```
Do we want to show the `devicePlugin`? 
What about the spaces (Line 23 and 25? Should they be removed?

Reply:
Changed "just like in...". No other changes.